src/AWS.Deploy.Orchestration/Orchestrator.cs

```diff
@@ -157,7 +158,7 @@ public Recommendation ApplyRecommendationPreviousSettings(Recommendation recomme
             {
                 if (previousSettings.TryGetValue(optionSetting.Id, out var value))
                 {
-                    _optionSettingHandler.SetOptionSettingValue(optionSetting, value);
+                   await  _optionSettingHandler.SetOptionSettingValue(optionSetting, value, skipValidation: true);
```

Comment:
I'm not too familiar with the previous setting code path, but is skipping validation here always appropriate? 

As validators become less deterministic (filesystem for Docker-related settings, network calls here), wondering if it's no longer safe to assume a previously-deployed option is still valid?

Reply:
I discussed this with Norm and he agreed that we don't need to run the validators when we get the values from CloudFormation, which is what this is doing. Setting values that we push up to CF, meaning a deployment was kicked off, will/should always be valid.